redaxo/src/addons/backup/lib/backup.php

```diff
@@ -300,7 +300,7 @@ public static function importFiles($filename)
      */
     public static function exportDb($filename, array $tables = null)
     {
-        $fp = @fopen($filename, 'w');
+        $fp = tmpfile();
```

Comment:
Ich meine, dass die Funktion in manchen Umgebungen nicht so auf Anhieb funktioniert, je nach Tempdir und Dateirechten.
Ggf. stattdessen mit einer normalen Datei als Tempdatei arbeiten, im Cache-Ordner, und die am Ende dann bloß an die richtige Stelle verschieben?